src/ray/common/scheduling/cluster_resource_scheduler.cc

```diff
@@ -590,6 +590,18 @@ std::string ClusterResourceScheduler::GetBestSchedulableNode(
   return string_to_int_map_.Get(node_id);
 }
 
+bool ClusterResourceScheduler::CheckIfSchedulable(
+    const std::unordered_map<std::string, double> &unit_resource) {
```

Comment:
Can we use `absl::flat_hash_map` whenever possible if there's no compatibility issue?